app/workloadSummaryReport/directives/workloadTable/workloadTable.html

```diff
@@ -0,0 +1,107 @@
+<div class="workload-table">
+	<div class="workload-table__container">
+		<table class="workload-table__table">
+			<colgroup>
+				<col style="width: 16%;">
+				<col style="width: 16%;">
+				<col style="width: 16%;">
+				<col style="width: 10%;">
+				<col style="width: 10%;">
+				<col style="width: 10%;">
+				<col style="width: 10%;">
+				<col style="width: 10%;">
+			</colgroup>
+			<tbody>
+				<tr>
+					<td class="workload-table__header-cell workload-table__cell-first">
+						Instructor
+					</td>
+					<td class="workload-table__header-cell">
+							Term
+					</td>
+					<td class="workload-table__header-cell">
+							Description
+					</td>
+					<td class="workload-table__header-cell">
+							Offering
+					</td>
+					<td class="workload-table__header-cell">
+							Enrollment
+					</td>
+					<td class="workload-table__header-cell">
+							Previous Enrollment
+					</td>
+					<td class="workload-table__header-cell">
+							Units
+					</td>
+					<td class="workload-table__header-cell workload-table__cell-last">
+							SCH
+					</td>
+				</tr>
+				<tr ng-repeat="instructor in instructors">
+					<td>
+						<div class="workload-table__cell workload-table__cell-first workload-table__cell-last-row">
+							{{ instructor.fullName }}
+						</div>
+					</td>
+					<td>
+						<div class="workload-table__value" ng-repeat="assignment in instructor.assignments | orderBy: 'term' ">
+							{{ assignment.term }} &nbsp;
```

Comment:
Describe the use of the non-breaking spaces here. Is this something that should be handled in CSS?